src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int thirdLeg;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int thirdLeg) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.thirdLeg = thirdLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (((firstLeg + secondLeg) / 2)
+                * (Math.sqrt(Math.pow((double) thirdLeg, 2))
```

Comment:
The formula for calculating the area of an isosceles trapezoid is incorrect. The correct formula is: ((topBase + bottomBase) / 2) * height. Update the formula accordingly.